app/routes/hydradx-ui/v1/stats/charts/tvl.mjs

```diff
@@ -23,7 +23,7 @@ export default async (fastify, opts) => {
         type: "object",
         properties: {
           asset: {
-            type: "integer",
+            type: "string",
             description: "Asset (id). Leave empty for all assets.",
```

Comment:
please update the description of all endpoints to reflect that you are expecting asset ticker here

Reply:
> please update the description of all endpoints to reflect that you are expecting asset ticker here

i don't expect asset ticker but asset id (e.g. '0')

Reply:
If your query expects it as string it's better to simply add `toString()`

Reply:
i also find it strange that 0 as integer breaks the query btw, there could be a solution on the query-level as well

Reply:
> If your query expects it as string it's better to simply add `toString()`
> 
> Doing it the other way around is hacky and it would skip validation (you can pass any string in params then)

add toString where? now all seems to be working fine

> i also find it strange that 0 as integer breaks the query btw, there could be a solution on the query-level as well

it no longer breaks the query, read the issue desc please

Reply:
The endpoint should accept param as integer (as asset_id is integer), and then convert it toString() before passing it to the query

Reply:
added toString conversion but that didn't help the problem, 0 is still wrong when accepted as integer as per issue description

Reply:
fixed the root cause